src/app/common/services/view-util.service.ts

```diff
@@ -0,0 +1,194 @@
+import {Injectable} from '@angular/core';
+import {AlfrescoApiService, LogService} from '@alfresco/adf-core';
+import {MinimalNodeEntity, RenditionEntry} from 'alfresco-js-api';
+import {ContentApiService} from '../../services/content-api.service';
+
+@Injectable()
+export class ViewUtilService {
```

Comment:
We have deprecated `common` folder in the development branch, please move the service to the traditional `app/services` folder instead.